package.json

```diff
@@ -24,18 +28,27 @@
   "jest": {
     "verbose": true
   },
+  "engines": {
+    "node": ">=8.7.0"
+  },
   "dependencies": {
-    "virtualfs": "^2.2.0"
+    "lodash": "^4.5.0",
```

Comment:
I'd prefer that we didn't use lodash. These utility libraries are intended for applications, not library authors themselves.